tools/SetupFlow/DevHome.SetupFlow/ViewModels/SummaryViewModel.cs

```diff
@@ -233,6 +270,14 @@ protected async override Task OnFirstNavigateToAsync()
         await ReloadCatalogsAsync();
     }
 
+    /// <summary>
+    /// Send telemetry about all next steps.
+    /// </summary>
+    private void ReportSummaryTaskCounts()
+    {
+        TelemetryFactory.Get<ITelemetry>().Log("Summary_NextSteps_Event", LogLevel.Critical, new CloneRepoNextStepsEvent(_cloneRepoNextSteps.Count), Orchestrator.ActivityId);

```

Comment:
>_cloneRepoNextSteps.Count [](http://example.com/codeflow?start=121&length=25)

nit: if you pass this into ReportSummaryTaskCounts as an argument, then this function just doesn't look like a weird blackbox that you have to step into to understand.

Reply:
Done.